src/ggt.ts

```diff
@@ -10,6 +10,11 @@ import { activeSpinner, spin } from "./services/output/spinner.js";
 import { installJsonExtensions } from "./services/util/json.js";
 
 export const ggt = async (ctx = Context.init({ name: "ggt" })): Promise<void> => {
+  if (process.env["GADGET_EDITOR_TERMINAL_SESSION_ID"]) {
+    output.writeStdout("Running ggt in the Gadget editor's terminal is not supported.\n");
```

Comment:
nit: `output` is a low level thing -- we can just use `println` here:
```suggestion
    println("Running ggt in the Gadget editor's terminal is not supported.");
```